collectioncardmaker.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class CollectionCardMaker {
```

Comment:
naming of the php file should be the same as this: CollectionCardMaker.php

check that all the require_once() and links etc match this